saw-remote-api/python/saw_client/crucible.py

```diff
@@ -524,6 +521,13 @@ def precondition(self, proposition : Union[str, CryptolTerm, cryptoltypes.Crypto
         else:
             raise Exception("preconditions must be specified before execute_func is called in the contract")
 
+    def precondition_f(self, s : str) -> None:
+        """Parses the given string like ``cry_f``, then calls ``precondition``
```

Comment:
Sort of minor, but I think this would be clearer as "Declares a precondition using a `cry_f`-style format string. I.e., `precondition_f(...)` is equivalent to `precondition(cry_f(...))`"